src/belgium/download.Rmd

```diff
@@ -0,0 +1,156 @@
+---
+title: "Download GBIF occurrences"
+author:
+- Damiano Oldoni
+- Peter Desmet
+date: "`r Sys.Date()`"
+output:
+  html_document:
+    toc: true
+    toc_depth: 3
+    toc_float: true
+    number_sections: true
+---
+
+This document describes how to retrieve occurrences for assessing the emerging status of alien species in Belgium.
+
+# Setup
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, warning = FALSE, message = FALSE)
+```
+
+Load libraries:
+
+```{r load_libraries}
+# Tidyverse packages
```

Comment:
I prefer the way we write the comments it in the checklist recipe: https://github.com/trias-project/checklist-recipe/blob/3e7491eec1437d15a2ff542163abd72796fddd52/src/dwc_mapping.Rmd#L31-L37

```
library(tidyverse)      # To do data science
library(lubridate)      # To work with dates
library(here)           # To find files
library(rgbif)          # To use GBIF services
library(trias)          # To use functions developed for TrIAS
```

Reply:
Fixed 14576b234e4dbce3dc08aed6cfa76ed28ed52b20